packages/evm/contracts/AragonProtocol.sol

```diff
@@ -115,9 +115,10 @@ contract AragonProtocol is Controller, IArbitrator {
     * @return subject Arbitrable instance associated to the dispute
     * @return ruling Ruling number computed for the given dispute
     */
-    function rule(uint256 _disputeId) external returns (IArbitrable subject, uint256 ruling) {
+    function rule(uint256 _disputeId) external returns (address subject, uint256 ruling) {
         IDisputeManager disputeManager = IDisputeManager(_getDisputeManager());
-        (subject, ruling) = disputeManager.computeRuling(_disputeId);
+        (IArbitrable _subject, uint8 _ruling) = disputeManager.computeRuling(_disputeId);
```

Comment:
Did we update the ruling options to be only uint8? TBH I think anything more than 2/4 options is going to need explicit documentation, so at that point, we might as well just use uint256 to save some gas 😄

Reply:
I think the main reason is to avoid having one extra slot when creating disputes